src/index.html

```diff
@@ -13,6 +13,48 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
In this case, we have duplication   `<div class="stars__star"></div>` better use `list` for it.